bin/publish

```diff
@@ -93,16 +93,15 @@ function update-exocom-package-versions
     process.exit 1
 
 
-function update-exocom-bus-number
+function update-exocom-dependency-number
   try
     replace.sync do
       files: ['features/**/*.feature', 'exosphere-shared/**/*.yml']
-      replace: /bus:(\s*)type: exocom(\s*)version: \d+.\d+.\d+/g
+      replace: /type: exocom(\s*)version: \d+.\d+.\d+/g
```

Comment:
We should probably also add a check for this now: https://github.com/Originate/exosphere/blob/12d35cdf89eabdbfff35bac237a67a0a464e11dd/exo-run/features/steps/steps-then.ls#L33